content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -11,29 +11,29 @@ Follow this guide to choose and set up a barcode scanner compatible with Odoo's
    :align: center
    :alt: An image of an example barcode scanner.
 
-   An image of an example barcode scanner.
-
 Scanner types
 =============
 
-Before setting up a barcode scanner, it is important to determine which scanner type best meets the
-needs of the business. There are three main types, each with their own benefits and use cases:
+Before setting up a barcode scanner, it is important to determine which type of scanner best meets
+the needs of the business. There are three main types of scanners that can be used with Odoo, each
+with their own benefits and use cases:
 
-- **USB scanners** are connected to a computer, and are suitable for businesses that scan products
-  at a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
-  compatible with the keyboard layout of the computer.
+- *USB scanners* are connected to a computer, and are suitable for businesses that scan products at
+  a fixed location, like at the checkout line in a grocery store.
 
-- **Bluetooth scanners** pair with a smartphone or tablet, making them an ideal cost-effective and
-  portable barcode scanner option. In this scenario, Odoo is installed on the smartphone, allowing
-  warehouse operators to handle operations, and check stock directly through their mobile devices.
+ .. important::
+    Ensure the chosen USB scanner is compatible with the keyboard layout of the computer.
```

Comment:
Admonition needs to be indented by _one more_ space to align with the text in the bullet point :)
```suggestion
  .. important::
     Ensure the chosen USB scanner is compatible with the keyboard layout of the computer.
```